content/zh/docs/contributing/_index.md

```diff
@@ -0,0 +1,552 @@
+
+- 改进现有内容。
+- 添加新的内容。
+- 更新 OpenTelemetry 注册表（Registry）。
+- 改进构建网站的代码。
+
+## 要求
+
+在贡献之前，你需要了解以下的技术和工具：
+
+- [git](https://git-scm.com/)
+- [GitHub](https://github.com/)
+- Markdown ([CommonMark](https://commonmark.org/))
+- YAML
+
+有关如何在本地构建和测试OpenTelemetry文档网站的技术细节，请参阅 [CONTRIBUTING.md](https://github.com/open-telemetry/opentelemetry.io/blob/main/CONTRIBUTING.md)文档。
+
+### 签署CNCF CLA {#sign-the-cla}
+
+所有 OpenTelemetry 贡献者**必须**阅读[贡献者指南](https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md) 并[签署贡献者授权同意书 (Contributor License Agreement, CLA)](https://docs.linuxfoundation.org/lfx/easycla/contributors)。
+
+未签署 CLA 的贡献者发起的拉取请求将无法通过自动化测试。你提供的姓名和邮箱必须与你的 `git config` 中的姓名和邮箱，并且你的 git 姓名和邮箱必须与 CLA 中使用的姓名和电子邮件相匹配。
+
+## 贡献新的内容
+
+```mermaid
+flowchart LR
+    subgraph first[如何贡献]
+    direction TB
+       T[ ] -.-
+       B[在GitHub克隆（fork）仓库] --- C[用markdown编写文档<br>并用Hugo构建网站]
+       C --- D[提交源文件到克隆的仓库]
+       D --- E[发起拉取请求（PR）]
+       E --- F[签署 CNCF CLA]
+    end
+
+classDef grey fill:#dddddd,stroke:#ffffff,stroke-width:px,color:#000000, font-size:15px;
+classDef white fill:#ffffff,stroke:#000,stroke-width:px,color:#000,font-weight:bold
+classDef spacewhite fill:#ffffff,stroke:#fff,stroke-width:0px,color:#000
+class A,B,C,D,E,F,G,H grey
+class S,T spacewhite
+class first,second white
+```
+
+_图 1. 贡献新的内容。_
+
+上图描述了如何贡献新的文档。
+
+要贡献新的内容页面或改进现有的内容页面，请发起拉取请求 (PR)：
+
+- 如果你的更改很小，或者你不太熟悉Git的使用，请阅读[使用Github进行更改](#changes-using-github) 来学习如何编辑一个页面。
+- 如果你的更改较大，请阅读[在本地分支进行更改](#fork-the-repo) 来学习如何在你电脑本地进行更改。
+
+{{% alert title="提醒" %}}
+
+可以将你的拉取请求变成草稿状态以表明内容未准备好接受评审。此时维护者仍可以发表评论或进行高级别的评审，但他们不会全面评审内容直到你删除草稿状态。
+
+{{% /alert %}}
+
+### 使用Github进行更改 {#changes-using-github}
+
+如果你对 Git 工作流程不太熟悉，这里有一种更简单的方法来发起拉取请求。图 2 概述了步骤，详细信息如下。
+
+```mermaid
+flowchart LR
+A([fa:fa-user 新贡献者]) --- id1[(open-telemetry或opentelemetry.io<br>GitHub仓库)]
+subgraph tasks[使用Github进行更改]
+direction TB
+    0[ ] -.-
+    1[1. 选择 Edit this page] --> 2[2. 使用GitHub markdown<br>编辑器进行编辑]
+    2 --> 3[3. 填写 Propose file change]
+
+end
+subgraph tasks2[ ]
+direction TB
+4[4. 选择 Propose file change] --> 5[5. 选择 Create pull request] --> 6[6. 填写 Open a pull request]
+6 --> 7[7. 选择 Create pull request]
+end
+
+id1 --> tasks --> tasks2
+
+classDef grey fill:#dddddd,stroke:#ffffff,stroke-width:px,color:#000000, font-size:15px;
+classDef white fill:#ffffff,stroke:#000,stroke-width:px,color:#000,font-weight:bold
+classDef k8s fill:#326ce5,stroke:#fff,stroke-width:1px,color:#fff;
+classDef spacewhite fill:#ffffff,stroke:#fff,stroke-width:0px,color:#000
+class A,1,2,3,4,5,6,7 grey
+class 0 spacewhite
+class tasks,tasks2 white
+class id1 k8s
+```
+
+_图 2. 使用Github发起拉取请求的流程。_
+
+1. 在发现问题的页面上，选择右侧导航面板中的**Edit this page**选项。
+2. 如果您不是该项目的成员，GitHub 会建议您创建存储库的分支。选择 **Fork this repository**。
+3. 在 GitHub 编辑器中进行更改。
+4. 填写**Propose file change**表格。
+5. 选择**Propose changes**。
+6. 选择**Create pull request**。
+7. 进入**Open a pull request**页面，填写描述可以帮助评审人理解你的更改内容。
+8. 选择**Create pull request**。
```

Comment:
```suggestion
4. 填写 **Propose file change** 表格。
5. 选择 **Propose changes**。
6. 选择 **Create pull request**。
7. 进入 **Open a pull request** 页面，填写描述可以帮助评审人理解你的更改内容。
8. 选择 **Create pull request**。
```

Reply:
Thanks for your reminder, I will recheck the whole document.